help/en/releasenotes/current-draft-note.shtml

```diff
@@ -101,7 +101,7 @@
 
         <h4>MQTT</h4>
             <ul>
-                <li></li>
+                <li>Fixed an issue in introduced in 5.5.5 where the incorrect default topics were used for Direction and Function.  The documenation was correct but the code wasn't.</li>
```

Comment:
@bobjacobsen 
It might be good to add a warning to the release notes of 5.5.5 that MQTT has wrong default topics. See the changes in this PR.